test/process_client.py

```diff
@@ -443,3 +444,16 @@ def get_source_key_from_metadata_comparison(md_comparison, url_prefix):
         k for k, v in md_comparison['metadata_sources'].items()
         if v['url'].startswith(url_prefix)
     ), None)
+
+
+def post_wfst(workspace, xml, *, headers=None, url=None):
```

Comment:
Make workspace optional with default value None. Assert workspace and url are not set together. If workspace and url are None, use `f"http://{settings.LAYMAN_SERVER_NAME}/geoserver/wfs?request=Transaction"`